### PR TITLE
[#15] Improve documentation

### LINKED ISSUE #15: Improve documentation
* Add CONTRIBUTING.md. Clearly describe the release process so that anyone in the future can contribute and release.
* Add CODEOWNERS
* Add CODE_OF_CONDUCT
* Add Pull Request template
* Add LICENSE and possibly acknowledgements for used dependencies
* Review package.json

### PR DESCRIPTION
**Pull request by ffeltrinelli**
### :pencil: Description

* Bumped Docusaurus from `2.0.0-alpha.72` to `2.0.0-beta.0`
* Added `CONTRIBUTING.md`
* Added `RELEASING.md`
* Added `CODEOWNERS`
* Added `CODE_OF_CONDUCT.md`
* Added `PULL_REQUEST_TEMPLATE.md`
* Added `LICENSE`
* Updated `README.md`

### :link: Related Issues
Fixes #15

### CODE REVIEW

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# See https://docs.github.com/en/github/creating-cloning-and-archiving-repositories/creating-a-repository-on-github/about-code-owners
+# as a reference on this file.
+# See https://git-scm.com/docs/gitignore#_pattern_format
+# as a reference on the filepatterns used.
+
+# Global Owners
+*       @ffeltrinelli @nikos912000 @lasanthak
```
Comment (nikos912000) [inline .github/CODEOWNERS]: I think we should create a GitHub Team for this :)

File: README.md [hunk 2]
```diff
@@ -20,7 +26,19 @@ npm start
...
+
+## ✏️ How to contribute
+
+Please see info in [CONTRIBUTING.md](./CONTRIBUTING.md).
+
+## ⚖️ License
 
-Any commit to `main` branch will trigger a GitHub Actions workflow that builds and releases
-the site to Github Pages.
+This repository is licensed under the [Apache License, Version 2.0](./LICENSE).
```
Comment (ffeltrinelli) [inline README.md:44]: @nikos912000 @lasanthak I'd like somehow to give credits / acknowledge we are using the free [Medium Pixel Point widget](https://medium-widget.pixelpoint.io/). Do you know if there is some standard for this kind of things (a file with a naming convention, etc)?

Conversely, I guess I don't need to give credits to all the libraries we depend on in the `package.json`, right?

Thanks!

Comment (nikos912000) [reply README.md:44]: I am not sure if there is a convention.
I have seen a `CREDITS.md` but also a `Credits` section in the readme.

**Review by nikos912000 — APPROVED** [review]

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,190 @@
...
+      License. However, in accepting such obligations, You may act only
+      on Your own behalf and on Your sole responsibility, not on behalf
+      of any other Contributor, and only if You agree to indemnify,
+      defend, and hold each Contributor harmless for any liability
+      incurred by, or claims asserted against, such Contributor by reason
+      of your accepting any such warranty or additional liability.
+
+   END OF TERMS AND CONDITIONS
+
+   Copyright 2019 Expedia, Inc
```
Comment (nikos912000) [inline LICENSE]:
```suggestion
   Copyright [yyyy] [name of copyright owner]
```

Comment (nikos912000) [reply LICENSE]: Afaik this needs to be the placeholder. You can copy-paste the one from [mittens](https://github.com/ExpediaGroup/mittens/blob/main/LICENSE).

Comment (ffeltrinelli) [reply LICENSE]: Thanks, copied!

Shall I also add a copyright notice on top of each source file of this repo?

File: .github/CODEOWNERS (hunk 1 above)
Comment (nikos912000) [inline .github/CODEOWNERS]:
```suggestion
*       @ExpediaGroup/oss-portal-admins
```

File: README.md (hunk 2 above)
Comment (lasanthak) [reply README.md:44]: I do not think you have to give credit for those projects that are being used, they are public and ppl should know. This feels like a special mention that a particular lib has played a major role.

But we can use a credits file or a section to mention some ppl who have contributed but it is not visible from the project (history, commits, or configs etc.).

In Jarviz, I have given credit to the others who sent PRs in changelog file as well. Not sure if that is necessary as it is obvious from commit history.

**Review by lasanthak — APPROVED** [review]

File: .github/CODEOWNERS (hunk 1 above)
Comment (ffeltrinelli) [reply .github/CODEOWNERS]: Thanks, replaced with team `oss-portal-admins`!

Comment (ffeltrinelli) [reply .github/CODEOWNERS]: Yes thanks, done!

File: README.md (hunk 2 above)
Comment (ffeltrinelli) [reply README.md:44]: Thanks both! I've added `CREDITS.md` in f4ee2e8d194012de51197a97b1e11e65f2dc8f4a , let me know if it looks good!

File: LICENSE (hunk 3 above)
Comment (ffeltrinelli) [reply LICENSE]: By looking at other `ExpediaGroup` projects the answer is yes, we need a copyright notice on top of source files. I've added it.

**Review by nikos912000 — APPROVED** [review]

**Pull request merged**